### PR TITLE
SuperAdmin functionality Part1 completed except TC3 -  Verify City in…

### LINKED ISSUE #3: SuperAdmin functionality Part 1
TC1: Verify super Admin  able to create Organization
1.Login as Super Admin
2.Click on Organization
3.Click on Add button
4.Enter Organization Name
5.Enter Organization Description
6.Select Organization Logo
7.Enter Address Line One
8.Enter Address Line Two
9.Enter Location
10.Enter City
11.Postal Code
12.Enter Contact Name
13.Enter Contact Number
14.Click On Submit
( * ) sign should be display for mandatory fields.
New Organization should be add

TC2: 
Verify super Admin  able to create Organization without mandatary fields

1.Login as Super Admin
2.Click on Organization
3.Click on Add button
4.Click on submit

Automated TC:
3. Verify City in Add Organization process
4. Verify Postal Code field in Add Organization process
5. Verify Contact Number in Add Organization process

### PR DESCRIPTION
**Pull request by rajaYendluri**
@pratik-9839  please review code

### CODE REVIEW

File: src/main/java/in/balendin/schoolmanagment/models/OrganizationData.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
+package in.balendin.schoolmanagment.models;
+
+import in.balendin.schoolmanagment.util.DataGenerator;
+
+public class OrganizationData {
+
+    private String Org_Title;
```
Comment (pratik-9839) [inline src/main/java/in/balendin/schoolmanagment/models/OrganizationData.java]: @rajaYendluri please use the camelcase for methods and variables

File: src/main/java/in/balendin/schoolmanagment/pages/AddOrganizationPage.java [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+    private WebElementFacade showEntries;
+
+    @FindBy(xpath = "//*[@id=\"OrganizationList_paginate\"]/ul/li[3]/a")
+    private WebElementFacade Pagination_2;
+
+    private OrganizationData data;
+
+
+    public void navigateToAddOrganizationForm() {
+        //  tab_Admin.click();
```
Comment (pratik-9839) [inline src/main/java/in/balendin/schoolmanagment/pages/AddOrganizationPage.java]: @rajaYendluri please remove the commented code

File: src/test/java/in/balendin/schoolmanagment/testcases/features/AddOrganizationTestCase.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+    @Before
+    public void setUp() {
+        loginSteps.navigateToApplication();
+        loginSteps.loginAsSuperAdmin();
+    }
+
+    @Test()
+    @Title("verify super admin can create Organization Successfully")
+    public void T1_addOrganization() {
```
Comment (pratik-9839) [inline src/test/java/in/balendin/schoolmanagment/testcases/features/AddOrganizationTestCase.java]: @rajaYendluri please remove the prefix name for every method name

File: src/main/java/in/balendin/schoolmanagment/pages/AddOrganizationPage.java [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+import in.balendin.schoolmanagment.models.OrganizationData;
+import net.serenitybdd.core.pages.WebElementFacade;
+import net.thucydides.core.pages.PageObject;
+import org.junit.Assert;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.support.FindBy;
+
+public class AddOrganizationPage extends PageObject {
+
+    @FindBy(xpath = "/html/body/div/aside/div/nav/ul/li[1]/ul/li[3]/a")
```
Comment (pratik-9839) [inline src/main/java/in/balendin/schoolmanagment/pages/AddOrganizationPage.java]: @rajaYendluri please use the relative XPath for form elements

Comment (pratik-9839) [inline src/main/java/in/balendin/schoolmanagment/pages/AddOrganizationPage.java]: @rajaYendluri Please  use the relative Xpath for all the form elements

**Review by pratik-9839 — CHANGES_REQUESTED** [review]

File: src/main/java/in/balendin/schoolmanagment/pages/AddOrganizationPage.java (hunk 2 above)
Comment (rajaYendluri) [reply src/main/java/in/balendin/schoolmanagment/pages/AddOrganizationPage.java]: completed

File: src/main/java/in/balendin/schoolmanagment/pages/AddOrganizationPage.java (hunk 4 above)
Comment (rajaYendluri) [reply src/main/java/in/balendin/schoolmanagment/pages/AddOrganizationPage.java]: completed

File: src/test/java/in/balendin/schoolmanagment/testcases/features/AddOrganizationTestCase.java (hunk 3 above)
Comment (rajaYendluri) [reply src/test/java/in/balendin/schoolmanagment/testcases/features/AddOrganizationTestCase.java]: completed

File: src/main/java/in/balendin/schoolmanagment/pages/AddOrganizationPage.java (hunk 4 above)
Comment (rajaYendluri) [reply src/main/java/in/balendin/schoolmanagment/pages/AddOrganizationPage.java]: completed

**Comment by rajaYendluri** [comment]
Organization page testcases and Processing Fee Testcases pushed

**Pull request merged**